docs/development/feature-flags.md

```diff
@@ -0,0 +1,100 @@
+# Feature Flags
+
+Feature Toggles (a.k.a [feature flags](https://martinfowler.com/articles/feature-toggles.html)) are a powerful way to change runtime behavior without changing the code itself. We encourage the use of the feature flags especially for preview features - explicit opt-in, and this document is focused on showing best practices/examples, how to configure and how they can be enabled when appropriate (i.e running e2e tests).
+
+![image](https://martinfowler.com/articles/feature-toggles/chart-4.png)
```

Comment:
I'd remove this since it may be copyrighted.

Reply:
Thanks for catching this @ItalyPaleAle.

> you're the license god for now on 💟